swap_meet/clothing.py

```diff
@@ -0,0 +1,11 @@
+from swap_meet.item import Item
+
+class Clothing(Item):
+    
+    def __init__(self, category="Clothing", condition=0):
+        self.category = category
+        self.condition = condition
```

Comment:
Consider using super. Category doesn't ever need to be passed in when instantiating a Clothing object since it will always be "clothing"
```suggestion
    def __init__(self, condition=0):
        super().__init__(category="Clothing", condition=condition)
```